docs/src/app/(public)/(content)/react/components/select/page.mdx

```diff
@@ -11,6 +11,7 @@
 ## Usage guidelines
 
 - **Prefer Combobox for large lists**: Select is not filterable, aside from basic keyboard typeahead functionality to find items by focusing and highlighting them. Prefer [Combobox](/react/components/combobox) instead of Select when the number of items is sufficiently large to warrant filtering.
+- **Special positioning behavior**: The select menu by default overlaps its trigger so the selected item's text is aligned with the trigger's value text. This behavior [can be disabled or customized](/react/components/select#positioning).
```

Comment:
```suggestion
- **Special positioning behavior**: The select popover by default overlaps its trigger so the selected item's text is aligned with the trigger's value text. This behavior [can be disabled or customized](/react/components/select#positioning).
```

Or "options list" or "popup" (as the component is called `Select.Popup`, I am afraid menu is a bit misleading as we also have a `Menu` component.